doc/explanation/clustering.md

```diff
@@ -161,19 +161,19 @@ However, you can control this behavior with the [`scheduler.instance`](cluster-m
 ### Instance placement scriptlet
 
 LXD supports using custom logic to control automatic instance placement by using an embedded script (scriptlet).
-This provides more flexibility than the built-in instance placement functionality.
+This method provides more flexibility than the built-in instance placement functionality.
 
 The instance placement scriptlet must be written in the [Starlark language](https://github.com/bazelbuild/starlark) (which is a subset of Python).
-The scriptlet will be invoked each time LXD needs to know where to place an instance.
-The scriptlet will be provided with information about the instance being placed, as well as providing information on the candidate cluster members that could host the instance.
+The scriptlet is invoked each time LXD needs to know where to place an instance.
+The scriptlet receives information about the instance that is being placed and the candidate cluster members that could host the instance.
```

Comment:
Is that correct? Or was it supposed to mean that the scriptlet provides info about candidates?